elements/bulbs-ellipsize/bulbs-ellipsize.js

```diff
@@ -0,0 +1,22 @@
+import { BulbsHTMLElement, registerElement } from 'bulbs-elements/register';
+import invariant from 'invariant';
+import truncate from 'truncate.js'; // eslint-disable-line no-unused-vars
+
+class BulbsEllipsize extends BulbsHTMLElement {
+  attachedCallback () {
+    invariant(this.getAttribute('line-count'),
+      'BulbsEllipsize: Expects a line-count attribute');
+    if (!this.textContent) { return; }
+
+    this.truncate();
+  }
+
+  truncate () {
+    let lineCount = parseInt(this.getAttribute('line-count'), 10);
+
+    $(this).truncate({ lines: lineCount, ellipsis: '...' });
```

Comment:
I'm not sure we should put jQuery in these elements. Looks like you can do this vanilla js style: [https://www.npmjs.com/package/truncate.js#as-a-vanilla-javascript-object](https://www.npmjs.com/package/truncate.js#as-a-vanilla-javascript-object)